app/models/advert.rb

```diff
@@ -124,11 +124,26 @@ def reader_postal_address
   end
 
   def self.buyer_of_options
-    ['Timber', 'Logs', 'Standing trees', 'Firewood']
+    'Timber
+    Logs
+    Standing trees
```

Comment:
That won't do; now you have two entries 'Standing' and 'trees', instead of one 'Standing trees'.
I'll incorporate your additions into my own commit..